lib/__init__.py

```diff
@@ -0,0 +1,19 @@
+__version__ = '1.1.0'
```

Comment:
It's necessary, it ensures that the dir is treated as a package.

* https://docs.python.org/3/tutorial/modules.html
* http://stackoverflow.com/questions/448271/what-is-init-py-for

In particular, if we want to export the renderers as a separate library (which I'm currently doing for my project, and I hope to extend for use with Napalm/Ansible/other tools), we need it.